src/scrabble.js

```diff
@@ -1,5 +1,51 @@
-function scrabble() {
-
+const lettervalue = {
+  a: 1,
+  e: 1,
+  i: 1,
+  o: 1,
+  u: 1,
+  l: 1,
+  n: 1,
+  r: 1,
+  s: 1,
+  t: 1,
+  d: 2,
+  g: 2,
+  b: 3,
+  c: 3,
+  m: 3,
+  p: 3,
+  f: 4,
+  h: 4,
+  v: 4,
+  w: 4,
+  y: 4,
+  k: 5,
+  j: 8,
+  x: 8,
+  q: 10,
+  z: 10
 }
 
+function scrabble(world) {
+  let sum = 0
+  let letter = ''
+  if (world === null) {
+    return 0
+  }
+  world = world.toLowerCase()
+  if (world === ' \t\n') {
+    return 0
+  }
+  if (world === '') {
+    return 0
+  }
+
+  for (let i = 0; i < world.length; i++) {
+    letter = world[i]
```

Comment:
Instead of assigning `world[i]` to a variable you could just use it directly in the line below, or maybe just declare the variable inside of the for loop to avoid possible problems since the variable is in a higher scope than it needs to be.